src/randoop/JunitFileWriter.java

```diff
@@ -5,12 +5,11 @@
 import java.io.PrintStream;
 import java.util.ArrayList;
 import java.util.Collections;
+import java.util.Iterator;
 import java.util.LinkedHashMap;
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
-import java.util.Set;
-
```

Comment:
I personally like to put a blank line between groups of import statements, and JDK imports feel different than imports from this package itself.  It doesn't matter to me either way, but let's keep the files consistent.